classes/Policies.php

```diff
@@ -225,6 +225,22 @@ class Policies {
         )
     );
 
+    private $db;
+
+    private $policy_id;
+
+    public function __construct($policy_id = NULL) {
+        $this->db = new \ParlDB;
+
+        if ( $policy_id ) {
+            $this->policy_id = $policy_id;
+        }
```

Comment:
It seems odd that if you pass in a policy_id, `getPolicies` still returns a list of all the policies; only `getArray` and `limitToSet` limit themselves. Should it be renamed `getAllPolicies`, or should this be fixed in the constructor as per previous comment.